syntax = "proto3";
package Tom;
 
// 订单请求参数
message OrderRequest {
    string orderId = 1;
    int64 timeStamp = 2;
}
 
// 订单信息
message OrderInfo {
    string OrderId = 1;
    string OrderName = 2;
    string OrderStatus =3;
}

 
enum gender_type
{
    MALE    = 0;
    FEMALE  = 1;
    OTHER   = 2;
}
 
message personal_info
{
    optional uint32         id = 1;
    optional string         name = 2;
    optional uint32         age = 3;
    optional gender_type    gender = 4;
    optional uint64         phone_num = 5;
}
 
message personal_info_list
{
    repeated personal_info  info = 1;
}